vNext/src/Microsoft.AspNet.OData/Routing/Conventions/DefaultODataRoutingConvention.cs

```diff
@@ -43,13 +43,36 @@ public ActionDescriptor SelectAction(RouteContext routeContext)
                     controllerName = entitySetSegment.EntitySet.Name;
                 }
 
+                var operationImportSegment = odataPath.FirstSegment as OperationImportSegment;
```

Comment:
Move this into a new class, such as OperationRoutingConvention?  Which controller will you route a function/action import to?

`operationImportSegment.EntitySet` can be null:
https://github.com/OData/odata.net/blob/master/src/Microsoft.OData.Core/UriParser/SemanticAst/OperationImportSegment.cs#L173-L180